benchmarks/continous/db/Justfile

```diff
@@ -0,0 +1,30 @@
+sql_dialect := "postgres"
+db_host := "34.90.190.128"
```

Comment:
Can you please add a comment on where we can find this IP in the future (if it ever changes)?

Reply:
Added a comment in [45b3366](https://github.com/near/nearcore/pull/11440/commits/45b336682425df8cf158a8643168e30f30c50b8d). I didn't see an indication of IP changes, so I assume it stays the same while the instance exists and the public IP is not disabled manually.